CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # tipi.build cli : CHANGELOG
 
+## v0.0.66 - Creative Cougar 🐾
+
+### Bug Fixes
+- Value given by the user's toolchain has priority over the toolchain generated by cmake-re
```

Comment:
replace with : 
- CMAKE_TOOLCHAIN_FILE content provided by the user can override cmake-re defaults like `CMAKE_FIND_ROOT_PATH_MODE_*`